### ISSUE TITLE
[1.2.16] - Backtrace errors 

### ISSUE THREAD

**Issue by bmfmancini** [Cacti/spine#183]
Hey Everyone,

I am seeing Spine backtraces without any noticeable issues polling and everything else works just fine
In this particular instance, the hosts are entered via DNS name vs IP  I am wondering if that's what the returned 14 address log is 

```
======= Backtrace: =========
*** Error in `/usr/local/spine/bin/spine': double free or corruption (out): 0x00007feae8012c20 ***
backtrace() returned 14 addresses
backtrace() returned 11 addresses
ffffffffff600000-ffffffffff601000 r-xp 00000000 00:00 0 [vsyscall]
7fff115a5000-7fff115a7000 r-xp 00000000 00:00 0 [vdso]
7fff1139e000-7fff1149f000 rw-p 00000000 00:00 0 [stack]
7f9c9c604000-7f9c9c605000 rw-p 00000000 00:00 0
7f9c9c603000-7f9c9c604000 rw-p 00022000 fd:00 145498 /usr/lib64/ld-2.17.so
7f9c9c602000-7f9c9c603000 r--p 00021000 fd:00 145498 /usr/lib64/ld-2.17.so
7f9c9c5ff000-7f9c9c602000 rw-p 00000000 00:00 0
7f9c9c5e9000-7f9c9c5f5000 rw-p 00000000 00:00 0
7f9c9c3e1000-7f9c9c403000 r-xp 00000000 fd:00 145498 /usr/lib64/ld-2.17.so
7f9c9c3ab000-7f9c9c3e1000 rw-p 00000000 00:00 0
7f9c9c3a9000-7f9c9c3ab000 rw-p 000cb000 fd:00 70979 /usr/lib64/libnetsnmp.so.31.0.2
7f9c9c3a6000-7f9c9c3a9000 r--p 000c8000 fd:00 70979 /usr/lib64/libnetsnmp.so.31.0.2
7f9c9c1a6000-7f9c9c3a6000 ---p 000c8000 fd:00 70979 /usr/lib64/libnetsnmp.so.31.0.2
7f9c9c0de000-7f9c9c1a6000 r-xp 00000000 fd:00 70979 /usr/lib64/libnetsnmp.so.31.0.2
7f9c9c0d4000-7f9c9c0de000 rw-p 00000000 00:00 0
```
NET-SNMP version: 5.7.2

**Comment by TheWitness** [comment]
Can you run a gdb on the core files and get a clean backtrace?  If you are not getting core files, setup your cron to allow them.  If you have questions on how to setup either logging cores, or setting a directory for those cores use DuckDockGo.

**Comment by bmfmancini** [comment]
Ok I think I narrowed down when it happens nothing to do with DNS or anything actually something on this installs side 
it occurs during a time when a bunch of devices become unreachable spine exceeds the polling time then the backtraces occur
So it must be something to do with the way spine handles a polling timeout

gdb didn't give me anything but I will get a fresh backtrace

**Comment by TheWitness** [comment]
Resolved in #184
